README.md

```diff
@@ -198,6 +198,7 @@ The table even leaves some room for custom encodings and is intended to work bot
   - `multibase` part of Kotlin project [multiformat](https://github.com/erwin-kok/multiformat)
 - [zig-multibase](https://github.com/zen-eth/multiformats-zig)
 - [Add yours here!](https://github.com/multiformats/multibase/edit/master/README.md)
+- `multibase` part of the [MultiformatsKit](https://github.com/ATProtoKit/MultiformatsKit) Swift package
```

Comment:
Same question as multicodec, should it be `Multibase`?

Reply:
Yes, it should be `Multibase`. My apologies for copying what everyone else was doing: I thought I had to because I thought there was a rule somewhere that it needed to be formatted that way.

I've updated the PR for you to take a look at.